src/admin/components/forms/field-types/RadioGroup/RadioInput/index.scss

```diff
@@ -45,6 +48,15 @@
     }
   }
 
+
+  &--is-focused {
```

Comment:
To target the neighboring element: `input[type=radio]:focus + .radio-input` and then apply the styles to that selector instead of setting a class.